### DISCUSSION TITLE
Android files modules

### DISCUSSION THREAD

**Discussion by rahimbtc** [mvt-project/mvt#309] [Q&A] [answered]
hello everyone;
can you explain the meaning of this "Found an SUID file in a non-standard directory" and how to confirm is either pegasus or not

**Comment by accesocero** [comment]
No es pot confirmar

**Comment by Manamama** [comment, marked as answer]
It depends. Some quick SUID theory:
https://www.tecmint.com/how-to-find-files-with-suid-and-sgid-permissions-in-linux/
